### DISCUSSION TITLE
Radio Paradise Bookmarks Ignore "Switch to Playback" Setting

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1624] [Fixed Issues]
@rern

This bug only shows when using RP bookmarks, not when selecting stations from the webradio library list.

When switching between RP bookmarks, the system momentarily tries to obey the " Switch to Playback on Add/Replace Play" setting, but then reverts to the library page. A QR code is sometimes visible for a fraction of a second.

This bug affects both Android and desktop Firefox.

Occasionally, it does switch to the Playback page, but I cannot see any pattern to this.

The bug is not present in 20241118.

**Comment by rern** [comment]
Please try:
`sed -i -E 's/.*(playerActive mpd) .* (playerStop)$/\1 \&\& radioStop || \2/' /srv/http/bash/cmd.sh`

**Reply by rern** [reply]
Just edited the command. Try again if already done it.

**Reply by R101G** [reply]
Thanks @rern, it switches to playback now, but there is no metadata.

**Reply by R101G** [reply]
... and sometimes it works for a while, and sometimes it stays on the library page.

**Comment by ceejayemm** [comment]
@rern 
I am getting this for ALL radio stations (after updating to 20240315 but may be caused by an earlier version ?).  Clicking on a radio station displays the options but, before I can select one of the options, the list disappears and the station entry is greyed out.  After that nothing, including the edit command given above, will allow me to select the radio station again. Rebooting the RPi resets the radio list but then the problem starts again.

![webradios](https://github.com/rern/rAudio/assets/25418025/a6abc303-8e9e-4bb7-9192-78b10aafd89c)

Chris

**Reply by rern** [reply]
Fixed by today's update.

**Reply by R101G** [reply]
@rern 

That fixes the menu problem, but using bookmarks to switch between RP stations still returns to the library page instead of the playback page.

I got this after the update:

/srv/http/bash/status.sh: line 495: ((: < 1000000 : syntax error: operand expected (error token is "< 1000000 ")
awk: cmd. line:1: BEGIN { printf "%.2f", /1000000 }
awk: cmd. line:1:                         ^ unterminated regexp
awk: cmd. line:1: BEGIN { printf "%.1f", /1000 }
awk: cmd. line:1:                         ^ unterminated regexp

**Reply by ceejayemm** [reply]
Update 20240316 fixes my issue with webradios.  I haven't seen the issue reported by @R101G above, if it helps I am running an RPi 3A for this purpose.

Thanks

Chris

**Comment by R101G** [comment]
@ceejayemm 
Is it the error message or the bookmarks problem you don't see?

**Comment by ceejayemm** [comment]
I tend not to use bookmarks, I don't see any coding errors when I apply the update. 

Chris

**Reply by R101G** [reply]
Thanks. I suspect I am the only user with RP bookmarks!

**Comment by R101G** [comment]
@rern
Webradio bookmarks bug is still present in 20240330

**Comment by R101G** [comment]
@rern 
I think the bookmarks bug might have been fixed by 20240406?

I do still get a QR code for http://eth0 displayed momentarily when changing channels, which seems strange. I use a wired connection with net.names disabled.
